File: Php/app/Urls/ForgotPasswordConfirmationUrlModel.php
namespace app\Urls{

	use app\RouteFormats;
	use app\Urls\BaseClasses\UrlModel;

	class ForgotPasswordConfirmationUrlModel extends UrlModel{

		public function __construct(){
			parent::__construct(RouteFormats::forgotPasswordConfirmation);
		}

	}

}

File: Web/Services/ForgotPasswordConfirmationUrlModel.cs
using Web.Models.UrlModels;
using Web.Routing;

namespace Web.Services
{
    public class ForgotPasswordConfirmationUrlModel : UrlModel
    {
        public ForgotPasswordConfirmationUrlModel()
            : base(RouteFormats.ForgotPasswordConfirmation)
        {
        }
    }
}